website/routes.py

```diff
@@ -173,6 +173,8 @@ def make_url_map(app):
 
     process_rules(app, [
 
+        Rule('/home/', 'get', website_views.home, OsfWebRenderer('home.mako')),
```

Comment:
When we say we need to maintain old routes does that mean "osf.io/" has to go to dashboard?

How about this:
"osf.io/" goes to what we call here "home.mako"  when logged in and index when not
"osf.io/myprojects" and "osf.io/dashboard"  goes to the "dashboard.mako"

If we just use "osf.io" for home we move away from calling it something like dashboard or home, which gives flexibility for future too. I don't know if there are technical issues about it.